$OpenBSD$

It limits the maximun size of the stack used by racket to the default size
imposed by OpenBSD to non-root users.

https://github.com/plt/racket/pull/316

--- src/racket/sconfig.h.orig	Sat Apr 13 22:57:50 2013
+++ src/racket/sconfig.h	Sat Apr 13 22:58:14 2013
@@ -297,14 +297,16 @@
 #  define SCHEME_PLATFORM_LIBRARY_SUBPATH "i386-openbsd"
 # endif
 
-/* Same reason as for FreeBSD? */
-# define ASSUME_FIXED_STACK_SIZE
-# define FIXED_STACK_SIZE 1048576
-
 # include "uconfig.h"
-# undef HAS_STANDARD_IOB
 
+# undef HAS_STANDARD_IOB
 # define HAS_BSD_IOB
+
+/* ASSUME_FIXED_STACK_SIZE works fine but OpenBSD supports variable stacksize */
+# undef UNIX_STACK_MAXIMUM
+# define UNIX_STACK_MAXIMUM 4194304
+# undef STACK_SAFETY_MARGIN
+# define STACK_SAFETY_MARGIN 100000
 
 #ifndef __ELF__
 # define UNDERSCORE_DYNLOAD_SYMBOL_PREFIX
